aws/resource_aws_pinpoint_app.go

```diff
@@ -128,6 +128,12 @@ func resourceAwsPinpointApp() *schema.Resource {
 					},
 				},
 			},
+			"arn": {
```

Comment:
This new attribute is missing documentation in `website/docs/r/pinpoint_app.html.markdown`, e.g. (under `## Attributes`)

```
* `arn` - Amazon Resource Name (ARN) of the PinPoint Application
```

Reply:
Added 👍